### PR TITLE
updated cli

### PR DESCRIPTION
**Pull request by shershah010**
If broker is not specified it will default to the streamer unless the streamer is not also a broker, then it will default to the paper broker.

### CODE REVIEW

File: harvest/cli.py [hunk 1]
```diff
@@ -15,6 +15,7 @@ def raise_helper():
     return raise_helper
 
 
+from harvest.utils import debugger
```
Comment (shershah010) [inline harvest/cli.py:18]: replace prints with debugger logs

File: harvest/cli.py [hunk 2]
```diff
@@ -97,7 +98,7 @@ def raise_helper():
 start_parser.add_argument(
     "-b",
     "--broker",
-    default="paper",
+    default="streamer",
```
Comment (shershah010) [inline harvest/cli.py:101]: replace default from paper to streamer

File: harvest/cli.py [hunk 3]
```diff
@@ -265,11 +267,26 @@ def _get_streamer(streamer):
...
+def _get_broker(broker, streamer, streamer_cls):
     """
-    Returns the storage instance specified by the user.
+    Returns the broker instance specified by the user.
     :broker: The type of broker to be instantiated.
     """
+
+    if broker == streamer:
+        debugger.info(f"Using streamer instance as broker. Streamer: {streamer}")
+        return streamer_cls
```
Comment (shershah010) [inline harvest/cli.py:278]: if the given broker is also the given streamer, then return the instantiated streamer class.

File: harvest/cli.py [hunk 4]
```diff
@@ -265,11 +267,26 @@ def _get_streamer(streamer):
...
+
+    if broker == "streamer":
+        if streamer in brokers.keys():
+            debugger.info(f"ℹ Using streamer instance as broker. Streamer: {streamer}")
+            return streamer_cls
+        else:
+            debugger.warning(
+                f"⚠ Streamer {streamer} does not support broker methods! Using paper broker."
+            )
+            return brokers.get("paper")()
```
Comment (shershah010) [inline harvest/cli.py:288]: If the broker is the default, then use the streamer. If the streamer is not a broker, use the paper broker.

**Review by tfukaza — APPROVED** [review]

**Pull request merged**